### DISCUSSION TITLE
webSocketServer example?

### DISCUSSION THREAD

**Discussion by xeroxstar** [webpack/webpack#14533] [Q&A] [answered]
Hello,

Is there an example for "./CustomServer" and "./CustomClient"? I cant figure out how to implement my custom socket with webSocketServer. 

```
module.exports = {
  //...
  devServer: {
    client: {
      webSocketTransport: require.resolve('./CustomClient'),
    },
    webSocketServer: require.resolve('./CustomServer'),
  },
};

```

**Comment by alexander-akait** [comment, marked as answer]
You can found example here https://github.com/webpack/webpack-dev-server/blob/master/client-src/clients/WebSocketClient.js

**Reply by alexander-akait** [reply]
And here https://github.com/webpack/webpack-dev-server/blob/master/lib/servers/WebsocketServer.js

**Reply by xeroxstar** [reply]
Thank you for quick reply :)

**Reply by angentle** [reply]
I tried using this demo,but it doesn't work. Have you successly run this demo?
